### PR TITLE
Added a LICENSE file

### PR DESCRIPTION
**Pull request by regislebrun**
…he OpenTURNS library.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The OpenTURNS library and all its derivatives are distributed under the LGPL license, see COPYING.
```
Comment (jschueller) [inline LICENSE]: actually it's COPYING and COPYING.LESSER files ; for the gpl and for the lgpl (lgpl is an extension of the gpl)

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,10 @@
+The OpenTURNS library and all its derivatives are distributed under the LGPL license, see COPYING and COPYING.LESSER.
+
+This library depends on different third-party codes, some of which being distributed under the GPL license. These dependencies are either mandatory or optional, but in all cases they are used with the explicit permission of their respective authors. All the details are in their respective COPYING files, and here is the list of the impacted dependencies:
++ the optional hmat library+
```
Comment (dbarbier) [inline LICENSE]: Remove trailing + sign here

**Comment by jschueller** [comment]
In the README.rst file we could change
"Please see the COPYING* files for details of the license of each components."
into
"Please see the LICENSE file for details."

**Comment by regislebrun** [comment]
@jschueller I kept the mention of the COPYING* files as the LICENSE file focuses only on GPL components and there are other components subject to a COPYING file.

**Pull request merged**